### PR TITLE
added 'npm' installing command

### PR DESCRIPTION
**Pull request by salmanfarisvp**
Microsoft azure is not comes with 'npm' and we continue 'sudo npm install -g node-red' without installing 'npm'

### CODE REVIEW

**Comment by knolleary** [comment]
Really? I would have expected the `curl -sL https://deb.nodesource.com/setup_4.x` line to install node including npm.

But you have highlighted these instructions still point at node 4.x and really must be updated for node 8

**Comment by salmanfarisvp** [comment]
I think that's not a problem , i just now tried with that ,and working fine.

**Comment by knolleary** [comment]
> I think that's not a problem 

Do you mean the node 4 vs node 8 issue? That is very much a problem - node 4 is no longer supported by node.js and we should not provide instructions that installs it.

**Comment by salmanfarisvp** [comment]
but for me it's working great  with azure server.. . anyway thanks for the information .

File: docs/platforms/azure.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ Once logged in you need to install node.js and Node-RED
 
        curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
        sudo apt-get install -y nodejs build-essential
+       sudo ap-get install npm
```
Comment (dceejay) [inline docs/platforms/azure.md]: what is ap-get :-)
This won't work as is - so may as well close...

Comment (salmanfarisvp) [reply docs/platforms/azure.md]: thanks , it's should be 'apt' , (changed now )

File: docs/platforms/azure.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Once logged in you need to install node.js and Node-RED
 
        curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
```
Comment (dceejay) [inline docs/platforms/azure.md]: really this should be 8.x now... - and that does include npm by default... so should not need the npm line

Comment (salmanfarisvp) [reply docs/platforms/azure.md]: Ok .

**Pull request closed without merging**